### PR TITLE
update ci; prep for publishing

### PR DESCRIPTION
**Pull request by devoncarew**
- update ci versions and add a dependabot config
- prep for publishing

### CODE REVIEW

File: pubspec.yaml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 name: source_maps
-version: 0.10.11-dev
+version: 0.10.11
 description: Library to programmatically manipulate source map files.
 repository: https://github.com/dart-lang/source_maps
```
Comment (kevmoo) [inline pubspec.yaml:5]: 👇  While you're at it, bump the min SDK to 2.18 – then use lints v2!!

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,4 +1,4 @@
-# 0.10.11-dev
+# 0.10.11
 
 - Switch to `package:lints`.
```
Comment (kevmoo) [inline CHANGELOG.md]: Not user facing. I'd drop this line!

Comment (devoncarew) [reply CHANGELOG.md]: done

File: pubspec.yaml (hunk 1 above)
Comment (devoncarew) [reply pubspec.yaml:5]: done

**Review by kevmoo — APPROVED** [review]

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,7 +1,6 @@
-# 0.10.11-dev
+# 0.10.11
 
-- Switch to `package:lints`.
```
Comment (kevmoo) [inline CHANGELOG.md:3]: 👍

**Pull request merged**